test/server/services/LocationService.spec.js

```diff
@@ -0,0 +1,68 @@
+import sinon from 'sinon';
+import { expect } from 'chai';
+import axios from 'axios';
+import MockAdapter from 'axios-mock-adapter';
+
+const LocationService = require('../../../server/services/location.js');
+let axiosMock;
+
+beforeEach(function(){
+    axiosMock = new MockAdapter(axios);
+});
+
+describe('LocationService', function() {
+
+    const locationService = new LocationService();
+
+    describe('findNearbyStores', function(){
+        let successfulResponse = {
+            "candidates": [
+               {
+                  "name": "Heinen's Grocery Store",
+                  "types": [
+                     "supermarket",
+                     "florist",
+                     "grocery_or_supermarket",
+                     "liquor_store",
+                     "food",
+                     "health",
+                     "point_of_interest",
+                     "store",
+                     "establishment"
+                  ]
+               }
+            ],
+            "status": "OK"
+        };
```

Comment:
Since the specs below directly modify the `successfulResponse` object, we should probably re-initialize it so there aren't any accidental side effects.

```suggestion
        let successfulResponse;
        
        beforeEach(() => {
            successfulResponse = {
                "candidates": [
                    {
                        "name": "Heinen's Grocery Store",
                        "types": [
                            "supermarket",
                            "florist",
                            "grocery_or_supermarket",
                            "liquor_store",
                            "food",
                            "health",
                            "point_of_interest",
                            "store",
                            "establishment"
                        ]
                    }
                ],
                "status": "OK"
            };
        });
```